automated-testing/container-testing/azdo-container-dev-test-release.md

```diff
@@ -0,0 +1,286 @@
+# Building Containers with Azure DevOps using DevTest Pattern
+
+In this documents, we highlight the learnings from the teams last project regarding how to apply the DevTest pattern to container development in Azure DevOps through pipelines.
```

Comment:
```suggestion
In this documents, we highlight learnings from applying the DevTest pattern to container development in Azure DevOps through pipelines.
```

Reply:
slight change suggested - while this "we", and "our last project", might have been relevant in a presentation or internal sharing doc, it sounds a bit weird here - after this PR is merged, who is we? - anytime we use the term we it should refer to all of CSE

Reply:
You are right, and I was not aware of that. thx.